src/broker/shutdown.c

```diff
@@ -66,19 +66,14 @@ static int forward_logbuf (flux_t *h,
     struct stdlog_header hdr;
     const char *txt;
     size_t txtlen;
-    char buf[FLUX_MAX_LOGBUF];
+    int len = strlen (stdlog);
+    char buf[len + 1];
```

Comment:
Since this is unbounded, are we concerned about blowing the stack if the system conjurs a really big log?

Reply:
You're correct, that case should be handled. I'm not sure if it would just be easier to just use the heap here, since this is a limited case. Otherwise, the VLA size could be replaced with a macro that takes the min of `len + 1` and some sane maximum (4K?) to avoid blowing up the stack?

Reply:
probably heap?  Otherwise it's the sane max just becomes the new FLUX_MAX_LOGBUF (albeit just in this case!)

Reply:
Ok, fixed that and addressed the other comments. Thanks!